CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ All user visible changes to this project will be documented in this file. This p
 
 - UI:
     - Redesigned media player. ([#1368], [#1367], [#1356])
+    - Set blockfolded whenever chat is favorite ([#1274], [#1391])
+        - User info page
+        - Group chat info page
+
+[#1274]: /../../issue/1274
+[#1391]: /../../pull/1391
```

Comment:
1) `/../../issue/1274` doesn't exist. Please, take a look at how GitHub's link to an issue look. Or scroll through this file.
2) References shouldn't be there.
3) CHANGELOG entry is a single entry, not a collection of entries formed like this.